moto/rds/models.py

```diff
@@ -1899,18 +1901,16 @@ def create_subnet_group(
         self.subnet_groups[subnet_name] = subnet_group
         return subnet_group
 
-    def describe_db_subnet_groups(
-        self, subnet_group_name: str
-    ) -> Iterable[SubnetGroup]:
+    def describe_db_subnet_groups(self, subnet_group_name: str) -> List[SubnetGroup]:
```

Comment:
If you want to avoid incurring the conversion to list on line 1910, this could probably be `describe_db_subnet_groups(self, subnet_group_name: Optional[str] = None)` and then I can type it with an overload.

Reply:
Nah, that's not necessary. I don't expect there to be ever more than 10 results, so the overhead for the list-conversion is negligible